packages/core/src/api/apis/definitions/errors.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ApiRef from '../ApiRef';
+
+/**
+ * Mirrors the javascript Error class, for the purpose of
+ * providing documentation and optional fields.
+ */
+export type Error = {
```

Comment:
Isn't it a bit unfortunate to call it exactly just Error like the global scope name?

Reply:
It's actually the point, I called it `ReportedError` at first but got a bit worried that people might be confused whether it would be ok to use a regular error. The type is exactly the same as the global scope one, but this way we can document some of the fields if needed.

Reply:
Yeah I'm just concerned about shadowing globals when importing, it's not very nice.

Reply:
Ah ye let's not expprt this type